spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,20 @@
+require 'beaker-rspec'
+require 'beaker-puppet'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
```

Comment:
do the acceptance tests also work with docker? If so, we should enable them for travis.

Reply:
Yes it works with docker.